File: frontend/Web/src/app/_shared/models/pessoa/endereco.ts
import { EnumEstadosBrasil } from "../estadosbrasil.enum";

export class Endereco {
    constructor(
        public cep?: number | string,
        public logradouro?: string,
        public numero?: number,
        public complemento?: string,
        public bairro?: string,
        public municipio?: string,
        public uf?: EnumEstadosBrasil
    ) {}
}

File: frontend/mobile/app/src/main/java/com/tecno/aging/domain/models/pessoa/Endereco.kt
package com.tecno.aging.domain.models.pessoa

data class Endereco(
    val cep: String = "",
    val logradouro: String = "",
    val numero: String = "",
    val complemento: String? = "",
    val bairro: String = "",
    val municipio: String = "",
    val uf: String = ""
) {
    constructor() : this("", "", "", "", "", "", "")
}